front/src/store/modules/recoveryPassword.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+const { serverPath } = require('@/config/config.dev.json');
+
+export default {
+  namespaced: true,
+  actions: {
+    recoveryPasswordSendEmail ({ commit }, payload) {
+      return axios.post(`${serverPath}/api/forgotPassword/`, { email: payload.email });
+    },
+    recoveryPasswordSendPassword ({ commit, state }, payload) {
+      return axios.post(`${serverPath}/api/forgotPassword/resetPassword/`, payload);
+    }
+  }
+};
```

Comment:
something wrong with style of file naming

Reply:
I don't understand, it's like I'm changing paths, but the "outdated" tag isn't added